### PR TITLE
Adds the saltmine shuttle.

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
add: Killing a revenant will now result in an unique shuttle to be able to be bought. You probably won't like it though.
add: Fake emag now available.
/:cl:

[why]: Inspired by a lot of salt lately, I have made a shuttle that contains more or less everything that makes people angry. Nothing that actually functions though, well except the clock cult room that's just unending skewers.

One of the chasms may be real.

Its based on the birdboat shuttle and flies sideways for memes
dreamseeker_2018-02-06_13-43-53

### CODE REVIEW

**Comment by Wilchenx** [comment]
>only one enternance
>ugly as fuck

i love it

**Comment by DaxDupont** [comment]
Ugly was an requirement.
jtmw95a 1

**Comment by Wilchenx** [comment]
you're missing something that gives you purrbation

**Comment by Supermichael777** [comment]
Stop Codegrif today; Vote YES on Proposition 521: Lobotomies for game developers.

**Comment by optimumtact** [comment]
you need to put your name on the changelog, or remove the `optional name here` descriptor to use your github name

**Comment by Supermichael777** [comment]
Wow its already annoying

**Comment by DaxDupont** [comment]
I'll fix the changelog tomorrow

**Comment by PsyCommando** [comment]
Needs more honk mechs, grey tide, cat people, furries, moths, cultists, revs, airlocks that crush you, bees, etc :P

**Comment by Armhulen** [comment]
haha another captain grief shuttle that people will use for a week before it gets forgotten

**Comment by DaedalusGame** [comment]
## DOES THIS SHUTTLE FLY SIDEWAYS

**Comment by DaxDupont** [comment]
Yes it does

**Comment by cebutris** [comment]
Might want to have the ratvar and narsie plushie not directly next to eachother. They don't really get along.

**Comment by octareenroon91** [comment]
>Might want to have the ratvar and narsie plushie not directly next to eachother. They don't really get along.

It's OK if you don't like it.

**Comment by DaxDupont** [comment]
made some changes

**Comment by DaxDupont** [comment]
Added a honkmech with barely an HP henk and guarded by clowns

**Comment by DaxDupont** [comment]
@Supermichael777 it's the donksoft without riot foam.

**Comment by Tlaltecuhtli** [comment]
pls stop with ooc in ic coding

**Comment by DaxDupont** [comment]
I haven't coded anything ooc in ic in this case.
It's a map!

**Comment by DaxDupont** [comment]
why is this do not merge

**Comment by Armhulen** [comment]
LOOOOOOOL GRIEF SHUTTLES

LOOOL THIS SHUTTLE'S GIMMICK IS THAT IT HAS EVERYTHING PEOPLE DON'T LIKE IN IT

File: _maps/shuttles/emergency_saltmine.dmm
Comment (Armhulen) [inline _maps/shuttles/emergency_saltmine.dmm]: THIS ONE WILL B A FAN FAVORITE LOOOOOOL

(you shouldn't place the salt and the real chasm on the same tile the salt will always fall into the chasm so it'll be like it didn't exist)

Comment (DaedalusGame) [reply _maps/shuttles/emergency_saltmine.dmm]: Are you sure about that

Comment (Armhulen) [reply _maps/shuttles/emergency_saltmine.dmm]: no

Comment (Armhulen) [reply _maps/shuttles/emergency_saltmine.dmm]: unless you're talking about this being a fan favorite, which in that case yes

**Comment by Armhulen** [comment]
plasma shuttle had a theme, had safe areas from the plasma flooring, and was 10000 credits. this is half the price, no theme beyond trying to put as much stupid garbage in the shuttle as possible and with two 200 hp clowns and a honk mech on the shuttle + 5 flashbangs just sitting around i'm sure you'll have trouble finding a safe spot from it

**Comment by DaedalusGame** [comment]
ITT: Contributor tries to design the designless

**Comment by Armhulen** [comment]
i don't want him to feel bad when i disable it in the config immediately after merge

**Comment by Armhulen** [comment]
and i didn't even mention that this shuttle turns into disco inferno anyways because you included plasma shuttles!

**Comment by epochayur** [comment]
include a crate with some printed-out goof PRs

**Comment by Qbopper** [comment]
garbage fire pr

**Comment by Iamgoofball** [comment]
re: salt falling in, he's right it will fall

**Comment by Iamgoofball** [comment]
re: salt falling in, he's right it will fall

**Comment by Iamgoofball** [comment]
@armhulen why would you disable it, dont be a dick :(

at least let the crew order it a few times

**Comment by ExcessiveUseOfCobblestone** [comment]
subtype of salt that refuses to fall in a chasm???

**Comment by ExcessiveUseOfCobblestone** [comment]
Replace all the turfs with sloth turfs for that extra fuck you.

**Comment by Poojawa** [comment]
Do you still slip on a high traction floor at wet = 2 ?

**Comment by DaxDupont** [comment]
I will fix the salt pile on the chasm,
I'll make the plasma statues inert.

Also the 2 200 hp clowns are there to defend the mech.
If someone goes through all the trouble of getting to that mech, it has 60 hp and the honker mech already has a -20 damage protection against melee.
It should break in two hits, hilariously the clowns fly into it with the new unbuckled merge and already damage it.

**Comment by DaxDupont** [comment]
But arm please don't complain about silly shit, all you PR is silly shit.

**Comment by optimumtact** [comment]
it's marked do not merge because I fully intend to close it at 24 hours because grief shuttles are not funny.

**Comment by DaxDupont** [comment]
Does this mean you're gonna remove the area, the supermatter shuttle and the daniel shuttle?
Those are more griff than a shuttle that's just slips, slips everywhere.

**Comment by DaxDupont** [comment]
The honk mech is as fragile as can be, There's hard to reach flashbangs, there's no weapons except the donk softones, of the non riot variarity.

All weapons you see are toys. The emag is fake.

**Comment by DaxDupont** [comment]
I am gonna lock it behind a revenant kill.

This will only be buyable after a revenant gets killed.

**Comment by optimumtact** [comment]
That's not a bad idea, I think as long it's rare then it will be fine

**Comment by YakumoChen** [comment]
It's actually a top tier shuttle because it comes with beacons which means you can teleport directly to it by default.

**Comment by AdamElTablawy** [comment]
>kill ghost with salt

>it gets revenge by giving you infinite salt

glorious

**Comment by Armhulen** [comment]
i'd much prefer a haunted revenant shuttle that requires you to "enchant the comms console" by scattering the dust over and it was actually a ghostly shuttle with floating candles and whispering walls but i can only be so mad and this works just fine so yeah sure let it live!

**Comment by DaedalusGame** [comment]
now i want ghost ship shuttle tho

**Comment by Armhulen** [comment]
i can make it if people want, the chairs will slowly float around and stuff. cool? cool!

File: _maps/shuttles/emergency_saltmine.dmm
Comment (ShizCalev) [inline _maps/shuttles/emergency_saltmine.dmm]: lmao

Comment (vuonojenmustaturska) [inline _maps/shuttles/emergency_saltmine.dmm]: No player references

**Review by vuonojenmustaturska — CHANGES_REQUESTED** [review]

**Comment by DaxDupont** [comment]
@ninjanomnom i fix

**Comment by WJohn** [comment]
why was this actually merged

**Pull request merged**
